enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts

```diff
@@ -0,0 +1,79 @@
+import { CONTAINER_NAME } from "./config";
+
+export const INSTALL_SDK_MESSAGE = `
+  Install the npm package on another terminal by running:
+  npm install --save @metabase/embedding-sdk-react
+`;
+
+export const DOCS_MESSAGE = `
+Metabase is running in a Docker container. To stop it, run "docker stop ${CONTAINER_NAME}".
+Documentation for the SDK can be found here: https://www.npmjs.im/@metabase/embedding-sdk-react
+`;
+
+export const ANONYMOUS_TRACKING_INFO = `
+By default, Metabase will phone home some data collected via Google Analytics and Snowplow.
+We don’t collect any usernames, emails, server IPs, database details of any kind, or any personally identifiable information (PII).
+
+This anonymous data helps us understand how people are actually using Metabase, which in turn helps us prioritize what to work on next.
+
+You can opt out by:
+
+1. Click on the gear icon.
+2. Select Admin settings.
+3. Go to the Settings tab.
+4. Click General
+5. Toggle the Anonymous tracking option.
+`;
+
+export const SDK_PACKAGE_NOT_FOUND_MESSAGE = `
+    The @metabase/embedding-sdk-react package is not installed.
+    
+    ${INSTALL_SDK_MESSAGE}
+    `;
+
+export const PACKAGE_JSON_NOT_FOUND_MESSAGE = `
+  Could not find a package.json file in the current directory.
+  Please run this command from the root of your project.
+`;
+
```

Comment:
@oisincoveney I think the indentation is different between the three messages (0 vs 2 vs 4 spaces), is this intended?

Reply:
oh absolutely not. I'll reformat everything - I think my editor is doing some weird things there